api/models/call_log_event.py

```diff
@@ -1,8 +1,18 @@
 from api.mixins import TimestampMixin
 from api import db
+from flask_sqlalchemy.query import Query as BaseQuery
+
+
+class CallLogEventQuery(BaseQuery):
+    def call_sid_exist(self, form_data):
+        return (
+            self.filter(CallLogEvent.call_sid == form_data["CallSid"]).first()
+            is not None
+        )
```

Comment:
Query to check whether the `call_sid` exists or not in the `call_log_event` table, the query is moved from the `event_service` to the `call_log_event` model.